packages/backend/src/links/links.controller.ts

```diff
@@ -1,34 +1,34 @@
-import { Controller, Get, Post, Body, Patch, Param, Delete } from '@nestjs/common';
-import { LinksService } from './links.service';
-import { CreateLinkDto } from './dto/create-link.dto';
-import { UpdateLinkDto } from './dto/update-link.dto';
+import { Controller, Get, Post, Body, Patch, Param, Delete, Query } from '@nestjs/common'
+import { LinksService } from './links.service'
+import { CreateLinkDto } from './dto/create-link.dto'
+import { UpdateLinkDto } from './dto/update-link.dto'
 
 @Controller('links')
 export class LinksController {
   constructor(private readonly linksService: LinksService) {}
 
   @Post()
   create(@Body() createLinkDto: CreateLinkDto) {
-    return this.linksService.create(createLinkDto);
+    return this.linksService.create(createLinkDto)
   }
 
   @Get()
-  findAll() {
-    return this.linksService.findAll();
+  findAll(@Query() params: any) {
```

Comment:
with `Query('title')` you can access just the value of the query parameter named 'title', and the type of that will be string. (Generally, it's good practice to avoid the any type whenever possible)

Reply:
The reason I wrote any is because of the future improvements on the queries. Because in the long run we should be able to filter by any property the link has shouldnt we?